tests/e2e/specs/backend/breadcrumbs.test.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import {
+	canvas,
+	createNewPost,
+	insertBlock,
+	switchUserToAdmin,
+} from '@wordpress/e2e-test-utils';
+import { searchForBlock } from '@wordpress/e2e-test-utils/build/inserter';
+
+/**
+ * Internal dependencies
+ */
+import {
+	filterCurrentBlocks,
+	goToSiteEditor,
+	useTheme,
+	waitForCanvas,
+} from '../../utils.js';
+
+const block = {
+	name: 'Store Breadcrumbs',
```

Comment:
After updating the block name it should be aligned here as well:

```suggestion
	name: 'WooCommerce Breadcrumbs',
```

Reply:
See ref. https://github.com/woocommerce/woocommerce-blocks/pull/8222#discussion_r1086850436